.gitignore

```diff
@@ -1,3 +1,4 @@
 composer.lock
 composer.phar
 vendor
+.idea
```

Comment:
please do not put developer environment specific stuff in the project .gitignore. to get this ignored you should use your global .gitignore. you can set it to your git client by using:

```
git config --global core.excludesfile ~/.gitignore_global
```